astro/src/content/docs/lifecycle/authenticate-users/oauth/endpoints.mdx

```diff
@@ -1290,7 +1197,7 @@ _Response Codes_
 {
   "client_id": "3c219e58-ed0e-4b18-ad48-f4f92793ae32",
```

Comment:
Should probably break these out to json files and include them. Not sure if appropriate to do now.